app/controllers/coronavirus_form/need_help_with_controller.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class CoronavirusForm::NeedHelpWithController < ApplicationController
-  before_action :check_first_question_answered
+  # before_action :check_first_question_answered
```

Comment:
```suggestion
```

This line should be deleted rather than commented out.

Reply:
:+1: